Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -3219,6 +3234,9 @@
 /* Title for Network Preference ssh config file selection dialog */
 "Please choose your ssh config files(s)" = "Please choose your ssh config files(s)";
 
+/* Title for Network Preference ssh known hosts file selection dialog */
+"Please choose your known_hosts file" = "Please choose your known_hosts file";
```

Comment:
nitpick - sometimes it's `known_hosts` and sometimes it's `known hosts` in the user-facing text. I don't have a preference between the two, but it would be good to be consistent :)